### PR TITLE
Display Power Control

### PR DESCRIPTION
**Pull request by WDobert**

### CODE REVIEW

**Comment by benjamin-owen** [comment]
What's difference between this and clearDisplay()? Just wondering

**Comment by alanssitis** [comment]
> What's difference between this and clearDisplay()? Just wondering

I think will is just adding a function to do this. Not sure if this is necessary but eh lgtm except for one change.

File: firmware-nucleo/smart-air-hockey-table/App/driver_display.c [hunk 1]
```diff
@@ -97,6 +91,14 @@ void Driver_Display_Clear(Display display)
 	transmit_end();
 }
 
+void Driver_Display_SetPowered(Display display, bool is_powered)
```
Comment (alanssitis) [inline firmware-nucleo/smart-air-hockey-table/App/driver_display.c:94]: Make this boi `inline`

**Review by alanssitis — CHANGES_REQUESTED** [review]
lgtm

**Comment by benjamin-owen** [comment]
@WDobert is there a point to including stdbool instead of just using an int?  Another just curious thing

**Review by alanssitis — APPROVED** [review]
:cowboy_hat_face:

**Pull request merged**